charge.go

```diff
@@ -91,6 +92,15 @@ type FraudDetails struct {
 	StripeReport FraudReport `json:"stripe_report"`
 }
 
+// Outcome is the charge's outcome that details whether a payment
+// was accepted and why.
+type Outcome struct {
```

Comment:
Ah yeah agreed, will change! I'll keep the named field on the `Charge` struct as `Outcome` since that's (I think) not as ambiguous.